code/modules/mining/abandoned_crates.dm

```diff
@@ -78,23 +79,23 @@
 			new /obj/item/clothing/suit/ianshirt(src)
 			new /obj/item/clothing/suit/hooded/ian_costume(src)
 		if(67 to 68)
-			new /obj/item/reagent_containers/glass/bottle/catmutation(src)
+			new /obj/item/toy/plush/awakenedplushie(src)
 		if(69 to 70)
 			new /obj/item/stack/ore/bluespace_crystal(src, 5)
 		if(71 to 72)
-			new /obj/item/reagent_containers/glass/bottle/lizardmutation(src)
+			new /obj/item/toy/plush/snakeplushie(src)
 		if(73 to 74)
-			new /obj/item/reagent_containers/glass/bottle/mothmutation(src)
+			new /mob/living/simple_animal/pet/gondola(src)
```

Comment:
gondolas can be killed and the meat can be ground for gondola mutation toxin I believe.